static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
So in our case, the require was dynamic, because a callback was passed. In other parts of the application, dynamic requires are being normalized everytime, so we assumed this one would be the same. However, this call did not succeed it's promise, until we tweaked the dynamic require from one that used the define(['require'], function(require){ require('module',cb) }) style to System.amdRequire('module',cb)